app/scenes/select_team/select_team.js

```diff
@@ -70,49 +71,105 @@ export default class SelectTeam extends PureComponent {
-                content.push(
-                    <Button
-                        key={team.id}
-                        onPress={() => this.onSelectTeam(team)}
-                        style={GlobalStyles.buttonListItemText}
-                        containerStyle={GlobalStyles.buttonListItem}
-                    >
-                        {team.display_name}
-                        <Icon
-                            name='keyboard-arrow-right'
-                            size={24}
-                            color='#777'
-                        />
-                    </Button>
-                );
-            }
-        }
+        const content = this.props.teams.map((team) => {
+            return (
+                <Button
+                    key={team.id}
+                    onPress={() => this.onSelectTeam(team)}
+                    style={style.buttonListItemText}
+                    containerStyle={style.buttonListItem}
+                >
+                    {team.display_name}
+                    <Icon
+                        name='keyboard-arrow-right'
+                        size={24}
+                        color='#777'
+                    />
+                </Button>
+            );
+        });
 
         return (
-            <View style={GlobalStyles.container}>
-                <Image
-                    style={GlobalStyles.logo}
-                    source={logo}
-                />
-                <Text style={GlobalStyles.header}>
-                    {this.props.config.SiteName}
-                </Text>
-                <FormattedText
-                    style={GlobalStyles.subheader}
-                    id='web.root.signup_info'
-                    defaultMessage='All team communication in one place, searchable and accessible anywhere'
-                />
-                <FormattedText
-                    style={GlobalStyles.subheader}
-                    id='signup_team.choose'
-                    defaultMessage='Your teams:'
-                />
-                {content}
+            <View style={style.container}>
+                <View style={style.wrapper}>
+                    <Image
+                        source={logo}
+                    />
+                    <Text style={style.header}>
+                        {this.props.config.SiteName}
+                    </Text>
+                    <FormattedText
+                        style={style.subheader}
+                        id='web.root.signup_info'
+                        defaultMessage='All team communication in one place, searchable and accessible anywhere'
+                    />
+                    <FormattedText
+                        style={[style.subheader, style.subheader2]}
+                        id='signup_team.choose'
+                        defaultMessage='Your teams:'
+                    />
+                </View>
+                <ScrollView style={style.scrollView}>
```

Comment:
yeah I do agree, but I just didn't want to scroll the whole logo and text.. but yeah I can do that pretty quick